applications/vanilla/models/class.categorymodel.php

```diff
@@ -3601,6 +3601,103 @@ private static function recalculateAggregateCounts() {
         self::instance()->clearCache();
     }
 
+    /**
+     * Recalculate  CountDiscussions and CountComments columns for category.
+     *
+     * @param int $categoryId Category ID to update
+     * @return void
+     */
+    public static function recalculateCategoryCounts(int $categoryId) {
```

Comment:
Since we're already in `CategoryModel`, we could probably make this a a more descriptive name without making it any longer. How about `recalculatePostCounts`?

Reply:
to me `Post` is very confusing in this context. But I do agree that method names are not very descriptive.